kolibri/core/assets/src/state/actions.js

```diff
@@ -187,14 +187,14 @@ function kolibriLogin(store, sessionPayload) {
 
 function kolibriLogout(store) {
   const coreApp = require('kolibri');
-  const { SessionResource, clearCaches } = coreApp.resources;
```

Comment:
Yeah - I don't know what this transforms to, but probably safest not to do it on non-require statements!